include/labwc.h

```diff
@@ -115,6 +115,14 @@ struct seat {
 		double x, y;
 	} smooth_scroll_offset;
 
+	/*
+	 * The surface whose keyboard focus is temporarily cleared with
+	 * seat_begin_server_action() and restored with
+	 * seat_finish_server_action().
+	 */
+	struct wlr_surface *last_focused_surface;
```

Comment:
Consider wrapping these two variables in a `struct { } seat_server_mode` or something like that for ease of reading.

Reply:
Good point. Wrapped with `focus_override`.